styles/all/template/js/cannedmessages_mcp.js

```diff
@@ -30,6 +30,15 @@ text_name = 'cannedmessage_content';
 		$(this).parents('li').remove();
 	});
 
+	// Remove emoji and other non-standard characters from title
+	document.getElementById('cannedmessage_name').addEventListener('input', function(e) {
+		const noEmoji = e.target.value.replace(/[\u{1F300}-\u{1F9FF}]|[\u{2700}-\u{27BF}]|[\u{2600}-\u{26FF}]|[\u{2300}-\u{23FF}]|[\u{1F000}-\u{1F6FF}]|[\u{2B00}-\u{2BFF}]/gu, '');
+
+		if (e.target.value !== noEmoji) {
+			e.target.value = noEmoji;
+		}
+	});
+
```

Comment:
Probably just do the same as for messages? Otherwise I assume there will be support requests about disappearing emojis from the title.

Reply:
There shouldnt be emoji in titles though. there's no reason. titles are for reference to moderators only of which canned message is.